proposals/issue_triage.md

```diff
@@ -16,21 +16,27 @@ The following are necessary and sufficient conditions for an issue to be conside
 
 * The issue must have a label indicating which one of the following kinds of issues it is
   
-  * bug
-  * question
-  * enhancement
-  * process
+  * **bug**
+    * Something is not working as intended in general.
+  * **question**
+    * Clear question statement
+    * Something is not working as intended in author's specific use case and he/she doesn't know why.
+  * **enhancement**
+    * everything is working as intended, but could be better (i.e more user friendly)
+  * **process**
 
```

Comment:
@elviraux Did you want to update this to provide some explanation of process?

Reply:
yes, as I pointed above, updating it right now.